### DISCUSSION TITLE
Newbie - Connection to server got closed. Server will not be restarted

### DISCUSSION THREAD

**Discussion by AdamCaretta** [microsoft/pylance-release#5256] [Q&A]
![Python Issue 2](https://github.com/microsoft/pylance-release/assets/80261632/cd0ea841-f65e-4fc8-88bd-57a66d882491)
![Python Issue](https://github.com/microsoft/pylance-release/assets/80261632/9bf9d514-dba7-4da3-b935-b4f1168c806a)

I saw on another post from years ago to post in this discussion for Python Extension issues. I am very new to all of this, and I'm not sure what's wrong here.

**Comment by rchiodo** [comment]
This is issue #5000, a known issue in the stable version. It's been fixed in the prerelease version.

**Comment by rchiodo** [comment]
The python extension is calling a function inside of our client that doesn't exist. We added it in the prerelease bits.
